hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx

```diff
@@ -27,6 +27,10 @@ import moment from 'moment';
 interface IAutoReloadPanelProps extends RouteComponentProps<object> {
   onReload: () => void;
   lastUpdated: number;
+  lastUpdatedOMDBDelta: number;
+  lastUpdatedOMDBFull: number;
+  lastUpdatedOMDBDeltaText: string;
+  lastUpdatedOMDBFullText: string;
```

Comment:
We can get rid of the variable here and just use hard-coded text right?

Reply:
Current Timestamp Can be from Full Snapshot or Delta Update so I have used  sort method in Descending order so If we do hardcoding of variables this logic will not apply so kept everything in separate variables.